R/Archiver.R

```diff
@@ -6,6 +6,9 @@ Archiver <- R6::R6Class( # nolint: object_name_linter.
     #' @description Returns an `Archiver` object.
     #'
     #' @return an `Archiver` object
+    #' @examples
+    #' archiver <- getFromNamespace("Archiver", "teal.reporter")
```

Comment:
Maintain class name when importing.
```suggestion
    #' Archiver <- getFromNamespace("Archiver", "teal.reporter")
```

Reply:
@chlebowa unsure if our lintr job will not update it afterwards to lowercase but will try

Reply:
If linter complains, kill the linter. The examples must present the object in its natural form.

Reply:
ok it didnt : ) will change where needed

Reply:
Hey, incorporated this update. I think it's way easier to track if we use class names. Thanks